### PR TITLE
Display inheritance information

### PR DESCRIPTION
**Pull request by gregelin**
Create `control.Inheritance` model and `control.Statement` foreign key to Inheritance to display inheritance information.

### CODE REVIEW

File: controls/models.py [hunk 1]
```diff
@@ -240,6 +241,18 @@ class StatementRemote(auto_prefetch.Model):
     import_record = auto_prefetch.ForeignKey(ImportRecord, related_name="import_record_statement_remotes", on_delete=models.CASCADE,
                                              unique=False, blank=True, null=True, help_text="The Import Record which created this record.")
 
+class Inheritance(models.Model):
+    name = models.CharField(max_length=120, unique=True, blank=False, null=False)
+    description = models.TextField(unique=False, blank=True, null=True)
+    responsibility = models.TextField(unique=False, blank=True, null=True)
+
+    def __str__(self):
+        return "'%s'" % (self.name)
```
Comment (alexanderward) [inline controls/models.py:250]: use F string:
`return f"{self.name}`

File: controls/models.py [hunk 2]
```diff
@@ -240,6 +241,18 @@ class StatementRemote(auto_prefetch.Model):
...
+
+    def __repr__(self):
+        # For debugging.
+        return "'%s'" % (self.name, self.id)
```
Comment (alexanderward) [inline controls/models.py:254]: use F String:
`return f"{self.name} - {self.id}"`

File: controls/views.py [hunk 3]
```diff
@@ -1328,13 +1328,16 @@ def component_library_component(request, element_id):
         # Get the impl_smts contributed by this component to system
         impl_smts = element.statements_produced.filter(statement_type=StatementTypeEnum.CONTROL_IMPLEMENTATION_PROTOTYPE.name)
 
+    inheritances = Inheritance.objects.all()
```
Comment (alexanderward) [inline controls/views.py:1331]: Should this always be all??  Or do we need a filter?

**Review by alexanderward — CHANGES_REQUESTED** [review]

File: controls/views.py (hunk 3 above)
Comment (gregelin) [reply controls/views.py:1331]: Yes. There are just three to five records in Inheritance, so really just using DB to allow content to change from install to install.

File: controls/models.py (hunk 2 above)
Comment (gregelin) [reply controls/models.py:254]: Updated.

File: controls/models.py (hunk 1 above)
Comment (gregelin) [reply controls/models.py:250]: Fixed

**Review by alexanderward — APPROVED** [review]

**Pull request merged**